src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.dao.impl.ProductDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private final ProductDao productDao = new ProductDaoImpl();
```

Comment:
Also I'd recommend initializing this dependency using the constructor. Please fix in all places